src/cnc/flavors/aws/shared/provision/partials/collection/dynamodb/dynamodb.tf.j2

```diff
@@ -1,4 +1,4 @@
-{% for environment in env_collection.active_environments %}
+{% for environment in env_collection.valid_environments %}
```

Comment:
Similar to what we do for rds, we should only keep the dynamodb resources around if `environment.active_deployment`

Reply:
solved